cmd/utils/utils.go

```diff
@@ -219,6 +219,14 @@ func MoveFile(source, destination string) error {
 		return errs.ErrCopyingEqualPaths
 	}
 
+	UnsetReadOnly(source)
+	UnsetReadOnly(destination)
+	tmpErr := os.RemoveAll(destination)
```

Comment:
Is the `destination` only **file path** or could it be the **folder path** as well?
`RemoveAll` could be dangerous to use here. Since `RemoveAll `removes the path and any children it contains.